app/dashboard/templates/profile_details.html

```diff
@@ -1,224 +1,182 @@
-                                                            {{b.title_or_desc | truncatechars:50}}
-                                                        </h6>
-                                                        <p>
-                                                            {{b.desc}}
-                                                        </p>
-                                                    </div>
-                                                    <div class="amount">
-                                                        <a class="btn btn-sm btn-darkBlue font-smaller-2">
-                                                            {{b.value_true}} {{b.token_name}},
-                                                            {{b.value_in_usdt}}
-                                                            USD
-                                                        </a>
-                                                    </div>
-                                                </a>
-                                            </li>
-                                        {% endfor %}
-                                    {% endif %}
-                                </ul>
+                                <h4>{% blocktrans %}{{profile.handle}} has Participated in these Funded Issues{% endblocktrans%}</h4>
+                                <div class="bounties text-center">
+                                    {% if bounties|length == 0 %} {% trans "No funded issues found" %} {% endif %} {% if bounties|length > 0 %}
+                                    <div class="row">
+                                        <div class="col-2 d-none d-sm-block">
+                                            <strong>{% trans "Relationship" %}</strong>
+                                        </div>
+                                        <div class="col-2">
+                                            <strong>{% trans "Status" %}</strong>
+                                        </div>
+                                        <div class="col-10 col-sm-8 col-md-4">
+                                            <strong>{% trans "Details" %}</strong>
+                                        </div>
+                                        <div class="col-4 col-md-4  d-none d-md-block">
+                                            <strong>{% trans "Amount" %}</strong>
+                                        </div>
+                                    </div>
+                                    {% for b in bounties %}
+                                    <div class="row bounty-row" data-ref-url="{{b.url}}">
+                                        <div class="col-2 d-none d-sm-block">
+                                            {% if profile.handle == b.fulfiller_github_username %} {% trans "Worker" %} {% elif profile.handle == b.bounty_owner_github_username%} {% trans "Funder" %} {% elif profile.handle == b.org_name %} {% trans "Repo
```

Comment:
Could we break this into multiple lines ! In one line I'm forced to scroll during code reviews :P .